docs/modify-volume.md

```diff
@@ -14,8 +14,8 @@ To use this feature, it must be enabled in the following places:
 - `VolumeAttributesClass` feature gate on `kube-apiserver` (consult your Kubernetes distro's documentation)
 - `storage.k8s.io/v1alpha1` enabled in `kube-apiserver` via [`runtime-config`](https://kubernetes.io/docs/tasks/administer-cluster/enable-disable-api/) (consult your Kubernetes distro's documentation)
 - `VolumeAttributesClass` feature gate on `kube-controller-manager` (consult your Kubernetes distro's documentation)
-- `VolumeAttributesClass` feature gate on `external-provisioner` (add `--feature-gates=VolumeAttributesClass=true` to `sidecars.provisioner.additionalArgs` when using the EBS CSI Helm chart)
-- `VolumeAttributesClass` feature gate on `kube-controller-manager` (add `--feature-gates=VolumeAttributesClass=true` to `sidecars.resizer.additionalArgs` when using the EBS CSI Helm chart)
+- `VolumeAttributesClass` feature gate on `external-provisioner` (automatically added on clusters compatible with `VolumeAttributesClass` for driver versions >= 1.35.0; otherwise add `--feature-gates=VolumeAttributesClass=true` to `sidecars.provisioner.additionalArgs` when using the EBS CSI Helm chart)
+- `VolumeAttributesClass` feature gate on `kube-controller-manager` (automatically added on clusters compatible with `VolumeAttributesClass` for driver versions >= 1.35.0; otherwise add `--feature-gates=VolumeAttributesClass=true` to `sidecars.resizer.additionalArgs` when using the EBS CSI Helm chart)
```

Comment:
I have no idea how we managed to screw this up and not include the docs changes, but can we instead take the more comprehensive and already approved changes from https://github.com/kubernetes-sigs/aws-ebs-csi-driver/pull/2114/files#diff-0120c51ecbff7d0c84fe341dc0ee5fdec550a3a54f871168c1d8a491f0234be2 that somehow didn't get un-reverted properly for `docs/modify-volume.md`?

Reply:
+1 to this it also addresses my above comment